app/src/main/java/com/sunlightlabs/congress/models/Legislator.java

```diff
@@ -31,7 +34,9 @@ public String titledName() {
         if (title == null) {
             // This will be wrong for Delegates and Resident Commissioners,
             // But I can live with that. In many contexts, we'll have the title.
-            if (chamber.equals("house"))
+			if (chamber == null)
+			    return getName();
```

Comment:
This change will cause members to show up title-less on joint committee screens.